HIS/his_checker.py

```diff
@@ -0,0 +1,138 @@
+#!/usr/bin/python3
```

Comment:
Also, try to be consistent in file naming. There is already one file name `license_check.py`. Maybe this one could be names `his_check.py`